monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -2597,7 +2597,7 @@ abstract class Observable[+A] extends Serializable { self =>
     *        throws an error.
     */
   final def onErrorRecover[B >: A](pf: PartialFunction[Throwable, B]): Observable[B] =
-    onErrorHandleWith(ex => (pf.andThen(Observable.now(_))).applyOrElse(ex, Observable.raiseError _))
+    onErrorHandleWith(ex => (pf.andThen(Observable.now)).applyOrElse(ex, Observable.raiseError))
```

Comment:
AFAIK it won't compile with latest `Scala 2.13` but it's fine unless you will have extra changes to do, I can fix it in PR updating to `2.13.0`